programs/hh-escrow/src/instructions/update_state.rs

```diff
@@ -0,0 +1,81 @@
+use anchor_lang::prelude::*;
```

Comment:
FYI: we'll need to change 'update_outcome' to 'update_state' instructions on the FE before deploying these changes?
Also, is this backwards compatible?

Reply:
This PR reverts a bunch of fixes and cleanup that I'd previously done, in favour of not changing the program that is currently deployed on mainnet.

Ideally we'd want to use the updated version, but Kyle wants to avoid breaking any existing markets.

Perhaps a solution would be to create a stripped down version of the fixes that is compatible with the existing accounts.

Reply:
Frontend will need to use `UpdateState`, it's a pain but it's being reverted to retain compatibility with the existing mainnet program.